utils/prepare-vagrant-dev.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# version 2 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+sudo apt-get install -q -y libcppunit-dev \
+          libspreadsheet-writeexcel-perl libdbd-sqlite3-perl \
+          openjdk-7-jre-headless
+
+./utils/prepare-test -afty
```

Comment:
If one calls this script this way, it breaks the current FOSSology instance without warning or asking. Please remove the `-a` flag.